src/utilities/utilities.ts

```diff
@@ -188,6 +188,29 @@ export function swiftpmSDKFlags(): string[] {
     return [];
 }
 
+/**
+ * Get build path flags to be passed to swift package manager and sourcekit-lsp server
+ */
+export function buildPathFlags(): string[] {
```

Comment:
Is it worthwhile combining this with the sdk flags functions. And instead have `configSwiftPMFlags` and `configSwiftDriverFlags` functions which are filled with swift build flags setup from the configuration.

@stevapple what's your thoughts. I know the `sdk` argument has special placement issues.

Reply:
> Is it worthwhile combining this with the sdk flags functions. And instead have configSwiftPMFlags and configSwiftDriverFlags functions which are filled with swift build flags setup from the configuration.

Agree on the initiative, but since some flags are unnecessary (eg. `--sdk` for `swift package`) it would be hard to do so.

Maybe better extracting these functions into `./utils/flags.ts`?

> I know the `sdk` argument has special placement issues.

I skimmed over the PR and think it won't break here.

Reply:
> Maybe better extracting these functions into ./utils/flags.ts?

Lets leave that for another PR

Reply:
We can leave this for now. When a PR for moving all the swift flags out to a separate file is done, tests can be added at that point.